test.js

```diff
@@ -83,6 +83,23 @@ test('exposes the internal pino', function (t) {
   logger.logger.info('hello world')
 })
 
+test('uses the log level passed in as an option', function (t) {
+  var dest = split(JSON.parse)
+  var logger = pinoHttp({ useLevel: 'debug', level: 'debug' }, dest)
+
+  function decoratedLogger (req, res, next) {
+    logger(req, res, next)
+    req.log.debug = function () {
+      t.end()
```

Comment:
can you please use the `on('data')` pattern in one of the other test and verify that `useLevel` is not included in the output?